sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregator/LongLastValueAggregator.java

```diff
@@ -64,24 +64,41 @@ public MetricData toMetricData(
       Map<Labels, Long> accumulationByLabels,
       long startEpochNanos,
       long epochNanos) {
-    List<LongPoint> points =
```

Comment:
is this a good change, to now have 3 places to make this call, rather than just the one?

Reply:
I think so, because in certain cases we do the conversion and not use it.